packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -123,19 +120,18 @@ const DomainPickerSuggestionItem: FunctionComponent< Props > = ( {
 				},
 				`type-${ type }`
 			) }
+			onClick={ onDomainSelect }
+			disabled={ isUnavailable }
 		>
-			{ [ ITEM_TYPE_RADIO, ITEM_TYPE_INDIVIDUAL_ITEM ].indexOf( type ) > -1 &&
+			{ type === ITEM_TYPE_RADIO &&
 				( isLoading ? (
 					<Spinner />
 				) : (
-					<input
-						aria-labelledby={ labelId }
-						className="domain-picker__suggestion-radio-button"
-						type="radio"
-						disabled={ isUnavailable }
-						name="domain-picker-suggestion-option"
-						onChange={ onDomainSelect }
-						checked={ selected && ! isUnavailable }
+					<span
+						className={ classnames( 'adomain-picker__suggestion-radio-circle', {
```

Comment:
Is there a reason we are using `adomain-picker__suggestion-radio-circle` and not `domain-picker__suggestion-radio-circle`?

Reply:
Definitely not. It's a mistake.